Framework/Core/src/SpyService.cxx

```diff
@@ -0,0 +1,40 @@
+//
```

Comment:
Please make sure you follow the coding convention and in particular that you put the copyright banner found in all the other files.